src/diffusers/models/model_loading_utils.py

```diff
@@ -128,7 +128,7 @@ def _fetch_remapped_cls_from_config(config, old_class):
         return old_class
 
 
-def load_state_dict(checkpoint_file: Union[str, os.PathLike], variant: Optional[str] = None):
+def load_state_dict(checkpoint_file: Union[str, os.PathLike], variant: Optional[str] = None, dduf_entries=None):
```

Comment:
```suggestion
def load_state_dict(checkpoint_file: Union[str, os.PathLike], variant: Optional[str] = None, dduf_entries: Optional[Dict[str, DDUFEntry]] = None):
```

Reply:
(nit) (unrelated to this PR) from what I can see, it doesn't seem that the `variant` parameter is ever used in the method